Formula/postgresql.rb

```diff
@@ -98,6 +98,9 @@ def post_install
     (var/"postgres").mkpath
     unless File.exist? "#{var}/postgres/PG_VERSION"
       system "#{bin}/initdb", "#{var}/postgres"
+      system "brew", "services", "start", "postgresql"
```

Comment:
Something like that seems superior if it still works if a server is already running (I suspect it won't, sadly). I think this may just be worth adding to the `caveats` if anything.